src/components/BoardHeader/BoardHeader.scss

```diff
@@ -59,7 +59,13 @@
   font-weight: bold;
   letter-spacing: 0.5px;
   line-height: 40px;
-  margin: 0px;
+  margin: 0;
+
+  text-overflow: ellipsis;
+  overflow: hidden;
```

Comment:
You could add `text-align: center;` here. And to avoid some offsets with the icon, we can add padding in line 84.

```
  .board-header__title {
    padding-right: 24px;
  }
 ```
 
![Screenshot 2021-10-03 at 17 32 38](https://user-images.githubusercontent.com/60005702/135760986-9ba63ee0-7544-451b-b2c8-6278c99ea2ae.png)

Reply:
Good catch! I forgot to remove the `width` prop and it wasn't centered anymore 👍